### DISCUSSION TITLE
Blender broken on Legacy Metal, Ventura 13.3

### DISCUSSION THREAD

**Discussion by ParaDoX1994** [dortania/OpenCore-Legacy-Patcher#1040] [Reports being investigated]
Errors with the following, reported first by user jhonjhon236 on Discord with MacBookPro9,2. After testing, able to reproduce on MacBookPro11,1 and MacBookPro11,5. No logs seem to have been created. 
OCLP version 0.6.2.

To repro: Launch Blender on a Legacy Metal machine.

<img width="372" alt="image" src="https://user-images.githubusercontent.com/35791009/228114231-87bb08d3-a59a-41bc-b388-5340a86224e9.png">

**Comment by khronokernel** [comment]
Verified on Macmini7,1, MTLCompiler seems to silently error. We see 31001 is in fact at fault, likely hinting to something dying in MTLCompiler (struct offsets changing perhaps?)
![Screenshot 2023-03-28 at 8 32 06 AM](https://user-images.githubusercontent.com/48863253/228272304-cc009010-5537-46db-b098-6658e5f6570f.png)
